cli/utils.js

```diff
@@ -196,11 +198,21 @@ const getCommandOptions = (pkg, argv) => {
     return { 'from-cli': true };
 };
 
-const done = errorMsg => {
-    if (errorMsg) {
-        logger.error(`${chalk.red.bold('ERROR!')} ${errorMsg}`);
+const doneFactory = successMsg => {
+    return errorMsg => {
+        if (errorMsg) {
+            logger.error(`${chalk.red.bold('ERROR!')} ${errorMsg}`);
+        } else if (successMsg) {
+            logger.info(chalk.green.bold(successMsg));
+        }
+    };
+};
+
+const printSuccess = () => {
+    if (process.exitCode === undefined || process.exitCode === 0) {
+        logger.info(chalk.green.bold(SUCCESS_MESSAGE));
     } else {
-        logger.info(chalk.green.bold('Congratulations, JHipster execution is complete!'));
+        logger.error(`${chalk.red.bold('ERROR!')} JHipster finished with code ${process.exitCode}`);
```

Comment:
Is this `${chalk.red.bold('ERROR!')}` necessary. The `logger.error` already highlights it in red; 

https://github.com/jhipster/generator-jhipster/blob/f40ebcc00f07d1ca1f7a45c0e7d57a158eb71c95/cli/utils.js#L47

Reply:
```suggestion
        logger.error(`ERROR! JHipster finished with code ${process.exitCode}`);
```